src/kitovu/sync/syncing.py

```diff
@@ -29,19 +30,23 @@ def _find_plugin(pluginname: str) -> syncplugin.AbstractSyncPlugin:
     plugin: syncplugin.AbstractSyncPlugin = manager.driver
     return plugin
 
-
 def start_all(config_file: typing.Optional[pathlib.Path]) -> None:
     """Sync all files with the given configuration file."""
     settings = Settings.from_yaml_file(config_file)
     for _plugin_key, connection_settings in sorted(settings.connections.items()):
         start(connection_settings)
 
+# synchronisation process: define here filename of FileCache / write with appdirs to disk
+
 
 def start(connection_settings: ConnectionSettings) -> None:
     """Sync files with the given plugin and username."""
     plugin = _find_plugin(connection_settings.plugin_name)
     plugin.configure(connection_settings.connection)
     plugin.connect()
+    filecache: kitovu_filecache.FileCache = kitovu_filecache.FileCache(pathlib.PurePath("filecache.json"))
+    # FIXME add path from settings instead of filecache.json
```

Comment:
You should be able to pass something like `pathlib.Path(appdirs.user_data_dir('kitovu')) / 'filecache.json'`.

(`Path` and not `PurePath` because it's a file on disk)